.github/workflows/pullrequest-stale.yml

```diff
@@ -0,0 +1,25 @@
+name: Stale
+on:
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  stale:
+    if: github.repository_owner == 'ls1intum'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 7 days stale PRs
```

Comment:
This name could be a bit more descriptive. E.g: "Check for stale PRs" or "Check for PRs with no activity for at least 7 days"

Reply:
Done!